doc/error.md

```diff
@@ -90,17 +90,32 @@ enum CompoundError {
     WithSource {
         source: Simple,
     },
+    #[from(ignore)]
+    WithBacktraceFromSource {
+        #[error(backtrace)]
+        source: Simple,
+    },
+    #[display(fmt = "{source}")]
+    WithDifferentBacktrace {
+        source: Simple,
+        backtrace: Backtrace,
+    },
     WithExplicitSource {
         #[error(source)]
         explicit_source: WithSource,
     },
+    #[from(ignore)]
+    WithBacktraceFromExplicitSource {
+        #[error(backtrace, source)]
+        explicit_source: WithSource,
+    },
     Tuple(WithExplicitSource),
     WithoutSource(#[error(not(source))] Tuple),
 }
 
 fn main() {
     assert!(Simple.source().is_none());
-    assert!(Simple.backtrace().is_none());
```

Comment:
The doc text above still mentions the backtrace method. This should be changed to talk about this `any::request_ref::<Backtrace>()` function